tools-and-tests/k6/data.json

```diff
@@ -1,7 +1,7 @@
 {
     "configs": [{
         "blockNodeUrl": "localhost:40840",
-        "protobufPath": "./../block-node-protobuf-0.23.1",
+        "protobufPath": "./../k6-proto",
```

Comment:
Note, the goal here was to match the current release proto folder name and then we'd automate the version bump every release so it always matched latest